p11-kit/test-objects.sh

```diff
@@ -212,3 +212,19 @@ else
 	sed 's/^/# /' list.diff
 	exit 1
 fi
+
+cat > list.exp <<EOF
+EOF
+
+"$abs_top_builddir"/p11-kit/p11-kit-testable generate-keypair -q --type=mock "pkcs11:" > list.out
+
+echo 7..7
```

Comment:
Maybe you might want to check #553 and possibly rebase on it; then you wouldn't need to maintain the number of tests by hand anymore.

Reply:
done